### ISSUE TITLE
Gift of Arthas and Retribution Aura

### ISSUE THREAD

**Issue by Awkakelen** [chromiecraft/chromiecraft#1574]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [X] 40-49
- [ ] 50-59

### Current Behaviour

Effect from "Gift of Arthas" is not included in retribution aura damage. It should deal 35~ damage
![obraz](https://user-images.githubusercontent.com/12753422/131233271-2ef787b8-ee4f-4bee-b165-8bc419b4449f.png)

### Expected Blizzlike Behaviour

It should stack with all damage, just as debuff says "increases their damage taken" 
![obraz](https://user-images.githubusercontent.com/12753422/131233281-a6bee2d2-86d4-4c62-a3f6-e47072575293.png)

### Steps to reproduce the problem

1. Get retribution aura
2. use gift of arthas potion
3. go to any mob
4. check damage

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/d20f315bd5370761f226460964ae506a97f372c0

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by Azcobu** [comment]
Thanks for the report. Firstly, I can confirm the report. Taking a Gift of Arthas elixir makes no change at all to the damage done by Ret Aura. It also doesn't affect Consecrate and a number of other pally spells that do Holy damage.

However, I think this is Blizz-like behaviour, and the main problem is just poor wording on the Gift tooltip. This is because the debuff that is actually cast on enemy mobs when the curse activates is on Wowhead here - https://tbc.wowhead.com/spell=11374/gift-of-arthas and if you look at the tooltip there, it says "Increases **physical** damage taken by 8 for 3 min." The physical damage qualifier is what's missing from the potion's tooltip, which gives the misleading impression that all damage is buffed, hence the confusion. And of course, Ret Aura does Holy damage, so it doesn't work with the Gift of Arthas.

This is confirmed by other comments on TBC - "The effect of this elixir only applies to physical attacks." ([source](https://tbc.wowhead.com/item=9088/gift-of-arthas#comments:id=149292:reply=22198)) and right below it someone noting it doesn't work with Consecrate either for the same reason.

So in summary, this appears to be Blizz-like, and it's just the Gift of Arthas tooltip is not accurately describing how the elixir actually works.
